### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: product-page/product-page.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      .filters-desc {
+        font-size: 1.5rem;
+        margin: 1.5rem;
+        cursor: pointer;
+      }
+      .filters-desc:hover {
+        text-decoration: underline;
+        color: 1rem solid bisque;
+      }
+      .flex-categories {
```
Comment (ckahub) [inline product-page/product-page.css:116]: use rem everywhere

File: product-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      }
+      .filter-desc-categories {
+        font-size: 1.7rem;
+      }
+      .filter-sorts {
+        margin: 8px;
+        font-size: 1rem;
+      }
+      .filter-categories {
+        margin: 0 5px;
```
Comment (ckahub) [inline product-page/product-page.css:128]: same

File: product-page/product-page.css [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      .txt-high-light {
+        color: #571089;
+        font-weight: 800;
+      }
+      
+      .grid-cards-icons {
+        margin: 5px;
+        width: 22rem;
+        border-radius: 5px;
+        border: 2px solid gray;
```
Comment (ckahub) [inline product-page/product-page.css:203]: use hex everywhere

**Review by ckahub — COMMENTED** [review]
do neccesary changes

File: product-page/product-page.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
+.nav-brand{
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    color: whitesmoke;
```
Comment (prakashsakari) [inline product-page/product-page.css:3]: use hex code for all the colors, use variables so that you can reuse it

**Review by prakashsakari — APPROVED** [review]
added a comment to change the color assignment to hex, rest looks good to me

**Pull request merged**